python/tank/constants.py

```diff
@@ -74,4 +74,7 @@
 ROOT_LOGGER_NAME = "tank"
```

Comment:
because of `__name__`, it needs to be `tank`. The option we have is to rename the main API to be `sgtk` rather than `tank`, which would be cool.